### PR TITLE
Release 0.2.0

### PR DESCRIPTION
**Pull request by BenFradet**
- [x] cla-bot
- [ ] ~~scala-steward~~
- [x] changelog rel date
- [x] final versions (~~iglu client and~~ schema ddl)

### CODE REVIEW

File: src/main/scala/com/snowplowanalytics/iglu/schemaddl/scalacheck/IgluSchemas.scala [hunk 1]
```diff
@@ -41,6 +41,6 @@ object IgluSchemas {
     schemaKey: SchemaKey
   ): F[Either[String, Json]] =
     resolver
-      .lookupSchema(schemaKey, 10)
+      .lookupSchema(schemaKey)
```
Comment (dilyand) [inline src/main/scala/com/snowplowanalytics/iglu/schemaddl/scalacheck/IgluSchemas.scala]: Is this related to changing the Scala version?

File: src/test/scala/com/snowplowanalytics/iglu/schemaddl/scalacheck/IgluSchemasSpec.scala [hunk 2]
```diff
@@ -54,7 +54,7 @@ object IgluSchemasSpec {
       .getOrElse(throw new RuntimeException("Invalid Iglu URI"))
 
     val result: EitherT[IO, String, (Gen[Json], Json)] = for {
-      r <- EitherT.right(Resolver.init[IO](0, Registry.IgluCentral))
+      r <- EitherT.right(Resolver.init[IO](0, None, Registry.IgluCentral))
```
Comment (dilyand) [inline src/test/scala/com/snowplowanalytics/iglu/schemaddl/scalacheck/IgluSchemasSpec.scala]: Is this related to changing the Scala version?

File: README.md [hunk 3]
```diff
@@ -4,48 +4,44 @@ ScalaCheck generators for various Iglu-compatible schema formats.
...
+libraryDependencies += "com.snowplowanalytics" %% "scalacheck-schema" % "0.2.0"
 ```
 
 ## Usage
 
 ```scala
-import org.json4s.jackson.parseJson
 import com.snowplowanalytics.iglu.schemaddl.jsonschema.Schema
-import com.snowplowanalytics.iglu.schemaddl.jsonschema.json4s.Json4sToSchema._
+import com.snowplowanalytics.iglu.schemaddl.jsonschema.circe.implicits._
```
Comment (dilyand) [inline README.md]: Maybe this should be moved to the `Fix readme examples` commit.

**Comment by alexanderdean** [comment]
Is there a plan to release this?

**Comment by dilyand** [comment]
To-do before release:

- [ ] Migrate to Github Actions.
- [ ] Amend c4dcacd to use `0.6.2` or later.
- [ ] Fix cf555fe commit message.
- [ ] Amend 8c122f9 to use `4.8.3` or later.
- [ ] Amend 287aba2 to use `1.4.5`.
- [ ] Amend f0e280c to extend copyright to 2021.
- [ ] Amend b6cefd7 to use `1.14.3` or later.
- [ ] Remove `0.2.0-M2` commit.
